sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -425,6 +425,46 @@ private static void AddBindingMetadata(IList<ExpandoObject> bindingMetadata, Cus
             bindingMetadata.Add(binding);
         }
 
+        /// <summary>
+        /// Creates a map of property name and it's binding property name, if specified from the attribute passed in.
+        /// </summary>
+        /// <remarks>
+        /// Example output generated from BlobTriggerAttribute.
+        ///     {
+        ///         "path" : "samples-workitems/{name}"
+        ///     }
+        /// 
+        /// In BlobTriggerAttribute type, the "BlobPath" property is decorated with "MetadataBindingPropertyName" attribute
+        /// where "path" is provided as the value to be used when generating metadata binding data, as shown below.
+        ///
+        ///     [MetadataBindingPropertyName("path")]
+        ///     public string BlobPath { set;get;}
```

Comment:
nit: very much not important but formatting/spacing here can be `{ set; get; }` instead

Reply:
Fixed.